packit/cli/builds/obs_build.py

```diff
@@ -0,0 +1,101 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+import os.path
+from tempfile import TemporaryDirectory
+from typing import Optional
+
+import click
+
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception, get_packit_api, iterate_packages
+from packit.config import get_context_settings, pass_config
+from packit.config.config import Config
+from packit.config.package_config import PackageConfig
+from packit.constants import (
+    PACKAGE_LONG_OPTION,
+    PACKAGE_OPTION_HELP,
+    PACKAGE_SHORT_OPTION,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@click.command("in-obs", context_settings=get_context_settings())
+@click.option(
+    "--owner",
+    help="OBS user, owner of the project. (defaults to the username from the oscrc)",
+)
+@click.option(
+    "--project",
+    help="Project name to build in. It will be created if does not exist."
+    " It defaults to home:$owner:packit:$pkg",
+)
+@click.option(
+    "--targets",
+    help="Comma separated list of chroots to build in. (defaults to 'fedora-rawhide-x86_64')",
+    default="fedora-rawhide-x86_64",
```

Comment:
What do you mean?

Reply:
Yes, the code assumes that you're talking to build.opensuse.org

Reply:
> What do you mean?

I just find it intriguing that the default target when building in OBS is _Fedora_ Rawhide instead of _openSUSE_ (Tumbleweed for example…)

Reply:
We picked the default of packit for consistency. ATM no one is using packit for openSUSE, so picking that as a default will ensure that the default won't be useful for anyone. We can change that later on